code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -81,23 +81,14 @@
 
 /obj/effect/spawner/lootdrop/prison_contraband
 	name = "prison contraband loot spawner"
-	loot = list(/obj/item/clothing/mask/cigarette/space_cigarette = 4,
-				/obj/item/clothing/mask/cigarette/robust = 2,
-				/obj/item/clothing/mask/cigarette/carp = 3,
-				/obj/item/clothing/mask/cigarette/uplift = 2,
-				/obj/item/clothing/mask/cigarette/dromedary = 3,
-				/obj/item/clothing/mask/cigarette/robustgold = 1,
-				/obj/item/storage/fancy/cigarettes/cigpack_uplift = 3,
-				/obj/item/storage/fancy/cigarettes = 3,
```

Comment:
would it be possible to turn these into their own loot pool, which you can then add as an entry in this loot list?

Reply:
Yeah, this is a good idea.